pkgs/by-name/ra/rainfuck-cli/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitLab
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rainfuck-cli";
+  version = "unstable-2024-01-20";
```

Comment:
I guess this is supposed to be 1.0.0? https://gitlab.com/Oglo12/rainfuck-cli/-/commit/dc9a885ef7daa4e4a60133c6114e582378978a85
Maybe ask upstream.

Reply:
Okay

Reply:
> Maybe ask upstream.

Did you? Upstream should create a tag.